packages/accordion/src/useAccordion.js

```diff
@@ -8,6 +8,19 @@
 import { useState } from 'react';
 import { composeEventHandlers, generateId, KEY_CODES } from '@zendeskgarden/container-utilities';
 
+const HOOK_ID = 'accordion';
+let PKG_VERSION;
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Rather than include this logic for every component, could we add in a global to the storybook webpack?

Since there is no root version we could default it to `[data-garden-container-version="storybook"] or something similar.

Reply:
yeah thats a great idea, don't really need to versions in storybook